web/themes/joinup/joinup_theme.theme

```diff
@@ -833,6 +834,30 @@ function joinup_theme_preprocess_page_title__node__discussion(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess_page_title__node__document().
```

Comment:
The hook is not really `hook_preprocess_page_title__node__document()` I believe

Reply:
Hmm, so its should be only Implements hook_preprocess_page_title() ?

Reply:
Most of the inherited hooks like this are https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/function/hook_preprocess_HOOK/8.2.x

Reply:
Ok, it is changed, but maybe we could create ticket to review and fix documentation in other preprocess functions.